.gitignore

```diff
@@ -107,3 +107,7 @@ dist
 
```

Comment:
zmian w głównych plikach będę dokonywał ja, bo powstają konfilkty. Ewentualnie poproś o dodanie nowego branchu. A to proszę usuń.

Reply:
no i mam tu jakiś problem bo jak odznaczę pliki których nie chce wysyłać to coś musze z nimi zrobić, i wrzucam je do ignore... masz jakiś inny pomysł na to ?

Reply:
wyłączyć watch sass i usunąć te pliki